apps/gpc-prover-client/app/hooks/useIdentity.ts

```diff
@@ -1,5 +1,5 @@
 import { useEffect, useState } from "react";
-import { Identity } from "semaphore-identity-v4";
+import { Identity } from "@semaphore-protocol/core";
```

Comment:
This works fine, though if you're only using Semaphore V4 (not also V3 like Zupass needs to) you can use @semaphore-protocol/identity which is smaller.
Most devs will never use V3, so can probably use the smaller one too, though the /core approach does seem to be the safest one we have for both right now.

Reply:
Good to know, changed it here: https://github.com/proofcarryingdata/zukyc/pull/27